docs/tests/approvals-tests.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Approvals Tests
+---
+
+# Approvals Tests
+
+Approvals are based on the idea of the golden master. You take a snapshot of an
+object and then compare all future versions of the object to the snapshot. A
+good use-case for Approvals tests is comparing rendered HTML.
```

Comment:
```suggestion
good use-case for the Approvals tests is to compare the rendered HTML to the test specified HTML.
```

I feel like when using the word compared you usually need to specify what to its being compared to.